handler/cantabular_metadata_export.go

```diff
@@ -135,13 +136,13 @@ func (h *CantabularMetadataExport) Handle(ctx context.Context, workerID int, msg
 				if *fd.IsAreaType {
 					areaTypeLabel = fd.Label
 					areaTypeName = fd.Name
-					areaTypeURL = fd.URI
+					areaTypeURL = "https://api.dp.aws.onsdigital.uk/v1/code-lists/" + strings.ToLower(fd.Name)
```

Comment:
Is this the correct value? Shouldn't the host be dynamic(like cfg.Host or something like that), I don't think the public API has `dp.` in it.